mmcv/utils/config.py

```diff
@@ -92,16 +93,43 @@ def _validate_py_syntax(filename):
                               f'file {filename}')
 
     @staticmethod
-    def _file2dict(filename):
+    def _substitute_predefined_vars(filename, temp_config_name):
+        fileDirname = osp.dirname(filename)
+        fileBasename = osp.basename(filename)
+        fileBasenameNoExtension = osp.splitext(fileBasename)[0]
+        fileExtname = osp.splitext(filename)[1]
```

Comment:
Although the predefined variables can be camel case, use snake case for python variables.